libsol/system_instruction.c

```diff
@@ -260,7 +260,13 @@ static int print_system_transfer_info(const SystemTransferInfo* info,
     SummaryItem* item;
 
     item = transaction_summary_primary_item();
-    summary_item_set_amount(item, "Transfer", info->lamports);
+
+    uint64_t  prioritization_fee = calculate_additional_transaction_fees();
+
+    // transaction_summary_reset makes sure that u64 value is 0 by default;
+    uint64_t lamports_total = info->lamports + prioritization_fee;
+
+    summary_item_set_amount(item, "Transfer", lamports_total);
```

Comment:
the fee should be displayed separately from the system transfer amount. remove this change